__tests__/unit/lib/utils/.forceignore

```diff
@@ -0,0 +1,10 @@
+# List files or directories below to ignore them when running force:source:push, force:source:pull, and force:source:status
+# More information: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_exclude_source.htm
+#
+
+package.xml
+**/jsconfig.json
+
+
+# LWC Jest
+**/__tests__/**
```

Comment:
I would prefer to move it under the __mocks__ folder 
(and adjust code reference to it in the repoGitDiff.test.js file)